nexus-writer/src/nexus/run.rs

```diff
@@ -0,0 +1,76 @@
+use super::{hdf5::RunFile, RunParameters};
+use crate::event_message::GenericEventMessage;
+use anyhow::Result;
+use chrono::{DateTime, Duration, Utc};
+use std::path::Path;
+use supermusr_streaming_types::ecs_6s4t_run_stop_generated::RunStop;
+
+#[derive(Debug)]
+pub(crate) struct Run {
+    parameters: RunParameters,
```

Comment:
Is there a reason for storing all the parameters?

Reply:
Possibly, not but I wanted to keep the flatbuffers code isolated from the hdf5 code as much as possible. So the RunParameters struct is the only interface between the two. I could place some of the run parameters into a new "RunHeader" object which would not be persisted. Is this worth doing do you think?